src/js/App/GlobalFilter/GlobalFilter.js

```diff
@@ -74,7 +74,7 @@ const GlobalFilterDropdown = ({
                   </ChipGroup>
                 ))}
                 {!isDisabled && (
-                  <Button variant="link" onClick={() => setValue(() => ({}))}>
+                  <Button variant="link" ouiaId="clear-global-filters" onClick={() => setValue(() => ({}))}>
```

Comment:
I think I would try to make all of these start the same for consistency and contain global-filter in some way. Here maybe `global-filter-clear`